src/seedu/addressbook/AddressBook.java

```diff
@@ -202,13 +202,17 @@ public static void main(String[] args) {
         showWelcomeMessage();
         processProgramArgs(args);
         loadDataFromStorage();
-        while (true) {
+        getUserInputContinuously();
+    }
+
+	private static void getUserInputContinuously() {
+		while (true) {
             String userCommand = getUserInput();
```

Comment:
Indentation problem here?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.